code/modules/mob/living/basic/lavaland/gutlunchers/gutlunchers.dm

```diff
@@ -0,0 +1,172 @@
+#define MAX_SIZE_CLAMP 1.1
+#define MAX_ATTACK_DIFFERENCE 3
+#define MAX_LOWER_ATTACK 15
+#define MINIMUM_POSSIBLE_SPEED 1
+#define MAX_POSSIBLE_HEALTH 100
+
+/mob/living/basic/mining/gutlunch
+	name = "gutlunch"
+	desc = "A scavenger that eats raw ores, often found alongside ash walkers. Produces a thick, nutritious milk."
+	icon = 'icons/mob/simple/lavaland/lavaland_monsters.dmi'
+	icon_state = "gutlunch"
+	icon_living = "gutlunch"
+	icon_dead = "gutlunch"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	basic_mob_flags = DEL_ON_DEATH
+	speak_emote = list("warbles", "quavers")
+	faction = list(FACTION_MINING, FACTION_ASHWALKER)
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "squishes"
+	response_harm_simple = "squish"
+	friendly_verb_continuous = "pinches"
+	friendly_verb_simple = "pinch"
+	gold_core_spawnable = FRIENDLY_SPAWN
+	death_message = "is pulped into bugmash."
+	///our size
+	var/size_scale
```

Comment:
Maybe a personal bugbear but I actually really dislike sprite scaling if it's not temporary, scaled sprites tend to introduce mixels and just look kind of bad.
Shrinking tends to look better than growing though so it's maybe fine for that if you don't have a child sprite.